client/src/components/Card.js

```diff
@@ -1,48 +1,207 @@
 import React, { Component } from 'react';
 import '../styles/Card.css';
+import {
+  Jumbotron,
+  Container,
+  Row,
+  Col,
+  Media
+} from 'reactstrap';
 
 class Card extends React.Component {
-	constructor(props) {
-		super(props);
-	}
-	
-	render() {
-		return (
-			// TODO: Remove card-parent-container
-			<div className="card-parent-container">
-			<div className="card-container">
-				<div className="name-mana-container">
-					<div className="name">
-						{this.props.name}
-					</div>
-					<div className="cost">
-						{this.props.cost}
-					</div>
-				</div>
-				<div className="image-container">
-					<img
-						className="card-image"
-						src={this.props.image}
-					/>
-				</div>
-				<div className="type-set-container">
-					<div className="type-container">
-						{this.props.type}
-					</div>
-					<div className="set-logo-container">
-						{this.props.set}
-					</div>
-				</div>
-				<div className="text-container">
-					{this.props.text}
-				</div>
-				<div className="power-toughness-container">
-					{this.props.power}/{this.props.toughness}
-				</div>
-			</div>
-			</div>
-		);
-	}
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <Container
+        fluid
+        className="card-container d-flex flex-column mh-100 h-100 mw-50 border rounded p-0 m-0"
+      >
+
+
+        {/* Card name and mana cost row */}
+        <Row
+          className="card-name-cost-row d-inline-flex no-gutters px-1 justify-content-between flex-grow-5 flex-shrink-1"
+          style={
+            {
+              "max-height": "15%"
+            }
+          }
+        >
+          {/* Card name */}
+          <Col
+            className="card-name-col px-0 flex-shrink-1 flex-grow-5 justify-content-start"
+          >
+            <p
+              style={
```

Comment:
I'd imagine it is not easy but we may want to consider refactoring these inline styles to some CSS.